frontend/packages/knative-plugin/src/components/pub-sub/PubSub.tsx

```diff
@@ -70,9 +72,10 @@ const PubSub: React.FC<PubSubProps> = ({
         close();
       })
       .catch((err) => {
+        const errMessage = err.message || 'An error occurred. Please try again';
         action.setStatus({
           subscriberAvailable: true,
-          error: err.message || 'An error occurred. Please try again',
+          error: t('knative-plugin-kn~{{errMessage}}', { errMessage }),
```

Comment:
I think it would be better to use localization function where string itself is defined:
```js
        const errMessage = err.message || t('knative-plugin-kn~An error occurred. Please try again');
```